### PR TITLE
Make identifiers a list instead of single value.

### PR DESCRIPTION
**Pull request by justinlittman**
refs https://github.com/sul-dlss/rialto-derivatives/issues/149

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,4 +1,14 @@
...
 module ApplicationHelper
+  def make_this_a_list(options: {})
+    options[:document] # the original document
+    options[:field] # the field to render
+    options[:value] # the value of the field
+
+    list = +'<ul>'
+    options[:value].each { |identifier| list << "<li>#{identifier}</li>" }
+    list << '</ul>'
+    list
```
Comment (jcoyne) [inline app/helpers/application_helper.rb]: Did you test this?  Does this work without it being marked `.html_safe`?

Comment (justinlittman) [reply app/helpers/application_helper.rb]: Yes, it does work without being marked `.html_safe`.
screen shot 2018-11-09 at 10 46 40 am

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,4 +1,14 @@
 # frozen_string_literal: true
 
 module ApplicationHelper
+  def make_this_a_list(options: {})
+    options[:document] # the original document
```
Comment (mjgiarlo) [inline app/helpers/application_helper.rb]: What are these three lines doing?

Comment (justinlittman) [reply app/helpers/application_helper.rb]: Hah, I copied from https://github.com/projectblacklight/blacklight/wiki/Configuration---Results-View#helper-method. I assumed they were doing some Ruby/Rails/Blacklight magic but maybe they were just demonstrating what was available in options ...

Comment (justinlittman) [reply app/helpers/application_helper.rb]: Yes, testing now.

Comment (justinlittman) [reply app/helpers/application_helper.rb]: @jcoyne Removed.

**Pull request merged**